### PR TITLE
Add cache control header

### PR DESCRIPTION
**Pull request by newsomc**

### CODE REVIEW

File: thumbor_aws/storages/s3_storage.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def put(self, path, bytes):
 
         file_key=Key(self.storage)
         file_key.key = file_abspath
-
+        file_key.metadata('Cache-Control', 'max-age=604800,public')
```
Comment (alpaker) [inline thumbor_aws/storages/s3_storage.py]: What I wrote in my email earlier was psuedo-code.  I think the syntax needs to be: `file_key.metadata['Cache-Control'] = 'max-age=604800,public'`.

Comment (newsomc) [inline thumbor_aws/storages/s3_storage.py]: Updated.

**Comment by alpaker** [comment]
+1

**Comment by alpaker** [comment]
After you merge i'll push this out to the running thumbor instance.

**Comment by newsomc** [comment]
Merged. Thanks @alpaker

**Pull request merged**